### PR TITLE
Stamen maps for spatial data

### PR DESCRIPTION
**Pull request by SwarnaBharathiMantena**
edav community contribution - Neelam Patodia, Kumari Nishu

### CODE REVIEW

File: Date_and_Time_Cheatsheet.Rmd [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Date and Time Cheatsheet in R programming Language 
+
+Kanika Aggarwal and Swarna Bharathi Mantena
+
+We both prepared a cheatsheet for the Date and Time manipulations in R programming language. Mentioned below is the github link and the references used. 
+NOTE: We uploaded the chestsheet as a pdf file in the github repository.
```
Comment (jtr13) [inline Date_and_Time_Cheatsheet.Rmd]: typo in cheatsheet ... please fix

File: Stamen_Maps_for_Spatial_Data.Rmd [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+Kumari Nishu and Neelam Patodia
+
+Objective: 
+We intend to highlight the usability of Stamen Maps for visualizing spatial data. 
+
+Approach:
+We conduct a comparative study between different types of graphs to understand the visualization tool which best demonstrates spatial data.We use the publicly available data on the number of vehicle collisions in New York for this demonstration.   
+
+
+```{r setup, include=FALSE}
```
Comment (jtr13) [inline Stamen_Maps_for_Spatial_Data.Rmd]: Please remove "setup" label from this chunk

File: Stamen_Maps_for_Spatial_Data.Rmd [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+library(caret)
+library(dplyr)
+library(ggmap)
+library(ggplot2)
+library(forwards)
+#library(plyr)
+library(gridExtra)
+
+#Loading the collisions dataset available from "https://data.cityofnewyork.us/Public-Safety/NYPD-Motor-Vehicle-Collisions-Crashes/h9gi-nx95/data"
+raw_df<-read.csv(file="/Users/neelampatodia/Downloads/collisions_data.csv", header=TRUE, sep=",")
```
Comment (jtr13) [inline Stamen_Maps_for_Spatial_Data.Rmd]: Please add `collisions_data.csv` to the resources folder and then change this link to that location.

**Comment by jtr13** [comment]
Please make the changes indicated above... thanks!

**Comment by SwarnaBharathiMantena** [comment]
Hello Professor,

We are using an original NYC collisions data which is of 355MB. As such as we had discussed with you after class, instead of uploading this file under resources, we thought of citing the original data source itself. Should we continue with this procedure or should we upload the data?
 Kindly suggest. 

Thanks,
Group 35

**Comment by jtr13** [comment]
We discussed a few options after class: 1) moving the analysis to your repo, 2) sampling to shrink the dataset, 3) read the data from Google drive.

File: Stamen_Maps_for_Spatial_Data.Rmd (hunk 3 above)
Comment (SwarnaBharathiMantena) [reply Stamen_Maps_for_Spatial_Data.Rmd]: A sample of the collisions data has been added to the resources and the link updated.

**Comment by jtr13** [comment]
Almost there... please fix the link where you read the data so that it reads from this repo, not yours. Thanks.

**Comment by SwarnaBharathiMantena** [comment]
Hello Professor,

The change has been made as suggested. Thank you.

**Comment by jtr13** [comment]
Great, thank you.

**Comment by jtr13** [comment]
I see what you mean about the sample not working well, so I changed the code to read the `.csv` directly from the URL. Also made some minor edits.

**Comment by jtr13** [comment]
🎉 Your pull request has been merged into master. Thanks for your contribution!

**Pull request merged**